README.md

```diff
@@ -33,6 +33,9 @@ the config parameter `temp_hysteresis`.
 To override these defaults, you can place a file at `/etc/zcfan.conf` with
 updated trip temperatures in degrees celsius. As an example:
 
+    max_level full-speed
```

Comment:
I'd place these below the `*_temp` entries since they are less likely to be regularly reconfigured, I guess?

Also maybe add a blank line between them.